packages/playwright-test/src/workerRunner.ts

```diff
@@ -179,6 +179,8 @@ export class WorkerRunner extends EventEmitter {
           entries.delete(tests[i]._id);
           await this._runTest(tests[i], entry.retry, tests[i + 1]);
         }
+      } else {
```

Comment:
- Why don't we report an error if any of the tests is missing, instead of when all the tests are missing?
- This should stop testing and trigger teardowns, similar to fatal error. See [this block](https://github.com/microsoft/playwright/blob/8c4fc62e8f601c81c16fc382f37999dfd4ce1213/packages/playwright-test/src/workerRunner.ts#L141-L146). Btw, perhaps we can just issue a fatal error instead of a new ipc?

Reply:
> Why don't we report an error if any of the tests is missing, instead of when all the tests are missing?

We could do that but it would require maintaining another set of not-found tests every time we perform filtering and the end result would be the same as currently the process converges with only unknown tests in the worker and we fail at that point. Given that it is very rare edge case let's keep the code simple.


> This should stop testing and trigger teardowns, similar to fatal error. See [this block](https://github.com/microsoft/playwright/blob/8c4fc62e8f601c81c16fc382f37999dfd4ce1213/packages/playwright-test/src/workerRunner.ts#L141-L146). Btw, perhaps we can just issue a fatal error instead of a new ipc?

Added `fatalUnknownTestIds` to `DonePayload` so that we can format test names in the runner (the worker doesn't know about those tests).